atf_core/src/atf_core/testblock.py

```diff
@@ -35,11 +34,11 @@ def get_result(self):
                 testblock_result.results.append(metric_result)
 
                 # aggregate result
-                if metric_result.groundtruth.result != None and not metric_result.groundtruth.result:
+                if metric_result.groundtruth.result != None and metric_result.groundtruth.result != Groundtruth.SUCCEEDED:
```

Comment:
I guess you can remove `metric_result.groundtruth.result != None`

Reply:
fixed in https://github.com/floweisshardt/atf/pull/150/commits/24ba26a2b83488bb61a08b9d2a9447f3da58d4ff